pandas/io/excel/_base.py

```diff
@@ -691,11 +698,28 @@ def __new__(cls, path, engine=None, **kwargs):
                     ext = "xlsx"
 
                 try:
-                    engine = config.get_option(f"io.excel.{ext}.writer")
+                    engine = config.get_option(f"io.excel.{ext}.writer", silent=True)
                     if engine == "auto":
                         engine = get_default_writer(ext)
                 except KeyError as err:
                     raise ValueError(f"No engine for filetype: '{ext}'") from err
+
+            if engine == "xlwt":
+                xls_config_engine = config.get_option(
+                    "io.excel.xls.writer", silent=True
+                )
+                # Don't warn a 2nd time if user has changed the default engine for xls
+                if xls_config_engine != "xlwt":
+                    warnings.warn(
+                        "The xlwt engine, the only engine that supports writing in "
+                        "an xls format, is no longer maintained and will be removed in "
+                        "a future version of pandas. Install openpyxl and write to an "
+                        "xlsx file instead. You can set the option "
+                        "io.excel.xls.writer to 'xlwt' to silence this warning.",
```

Comment:
But setting the option is also deprecated and will also raise a warning?

Reply:
Yes - but at least it is in hopefully fewer places in user code. The warning can still be ignored there, but I see now that will cause confusion when a user follows the instruction and then still gets a warning. I'm thinking I'll change to:

> You can set the option io.excel.xls.writer to 'xlwt' to silence this warning. This option is deprecated as well and will also give a warning, but can be globally set and suppressed.